tests/functional/behave_features/HC-14_user_submits_chart_with_errors.feature

```diff
@@ -37,7 +37,8 @@ Feature: Chart submission with errors
     Examples:
       | vendor_type  | vendor    | chart_path                     | message                               | bad_version | 
       | partners     | hashicorp | tests/data/vault-0.17.0.tgz    | doesn't match the directory structure | 9.9.9       |
-    
+      | partners     | hashicorp | tests/data/vault-0.17.0.tgz    | The chart verifier returned an error  | abc-9.9.9       |
+
```

Comment:
Is this test resolving https://issues.redhat.com/browse/HELM-409 or do we need to any other test